donut/modules/voting/helpers.py

```diff
@@ -412,5 +431,5 @@ def get_results(survey_id):
         elif question_type == question_types['Checkboxes']:
             question['results'] = Counter(chain(*responses)).most_common()
         elif question_type == question_types['Elected position']:
-            question['results'] = winners(responses)
+            question['results'] = winners(responses)[:WINNERS_TO_LIST]
```

Comment:
Yeah I was worried that people would write in the entire Bee Movie script. But I guess that's not a problem any more since they can only write in Donut users.